Sources/KeyboardShortcuts/Localization/zh-Hans.lproj/Localizable.strings

```diff
@@ -0,0 +1,6 @@
+"Record Shortcut" = "记录快捷键";
+"Press Shortcut" = "键入快捷键";
+"This keyboard shortcut cannot be used as it's already used by the" = "当前快捷键无法使用，因为它已用作菜单项";
+"menu item." = "的快捷键。";
+"This keyboard shortcut cannot be used as it's already a system-wide keyboard shortcut." = "当前快捷键无法使用，因为它已用作系统快捷键。";
+"Most system-wide keyboard shortcuts can be changed in “System Preferences › Keyboard › Shortcuts“." = "可以在 “系统偏好设置 › 键盘 › 快捷键” 中更改大多数系统快捷键。";
```

Comment:
Do you know anyone that knows Chinese that can review your translation?

Reply:
Actually I am Chinese. :p

Reply:
Yes, I'm not doubting that. But it's always helpful to have someone else look over the translation to spot typos, etc. I would ask the same for any other language.

Reply:
Yeah sure, I will let my friends have a look.

Reply:
@sindresorhus Chinese is my native language and I think this is the proper translation.

Reply:
Thanks for reviewing :)

Reply:
@sindresorhus Used a single string, please have a look if it is OK.

Reply:
@sindresorhus I think this translation is just right,well done